application/common/models/Method.php

```diff
@@ -25,10 +25,33 @@ public function rules()
                 ],
 
                 [
+                    ['verified', 'verification_attempts'], 'default', 'value' => 0,
+                ],
+
+                [
+                    'verification_code', 'default', 'when' => function() { return $this->getIsNewRecord(); },
```

Comment:
Does Yii2 to require you to use this `'when' => function() { return $this->getIsNewRecord(); }` syntax rather than some sort of `'on' => 'insert'` syntax the way Yii1 worked?